README.md

```diff
@@ -17,6 +17,104 @@ You can install `SymEngine.jl` by giving the following command.
 julia> Pkg.add("SymEngine")
 ```
 
+## Quick Start
+
+### Working with scalar variables
+
+#### Defining variables 
+One can define variables in a few ways the following 3 examples are equivent.
```

Comment:
We should add a whitespace here. Also this can be changed to "... in a few ways. The following three examples are equivalent:"